lib/internal/crypto/rsa.js

```diff
@@ -306,35 +308,36 @@ async function rsaImportKey(
   }, keyUsages, extractable);
 }
 
-function rsaSignVerify(key, data, { saltLength }, signature) {
-  let padding;
-  if (key.algorithm.name === 'RSA-PSS') {
-    padding = RSA_PKCS1_PSS_PADDING;
-    // TODO(@jasnell): Validate maximum size of saltLength
-    // based on the key size:
-    //   Math.ceil((keySizeInBits - 1)/8) - digestSizeInBytes - 2
-    validateInt32(saltLength, 'algorithm.saltLength', -2);
-  }
-
+async function rsaSignVerify(key, data, { saltLength }, signature) {
   const mode = signature === undefined ? kSignJobModeSign : kSignJobModeVerify;
   const type = mode === kSignJobModeSign ? 'private' : 'public';
 
   if (key.type !== type)
     throw lazyDOMException(`Key must be a ${type} key`, 'InvalidAccessError');
```

Comment:
The error is not sync anymore. Does this make this semver-major change?

Reply:
It does not. The entire API surface of webcrypto is async functions.